laterpay/asset_sources/js/laterpay-backend.js

```diff
@@ -47,8 +48,38 @@ function removeLoadingIndicator ($target) {
         $target.removeClass('lp_is-delayed');
     } else {
         // remove the loading indicator
-        $target.find('.lp_loadingIndicator').remove();
+        $target.find('.lp_loading-indicator').remove();
```

Comment:
Since it is used as a JS hook, it should also have a JS class -> '.lp_js_loadingIndicator'. '.lp_loading-indicator' is a class for applying CSS styling.